### ISSUE TITLE
AddCommasToMatrices=0 doesn't affect some matrices

### ISSUE THREAD

**Issue by homocomputeris** [davidvarga/MBeautifier#25]
Hi!
`[[I, Z]; [h * (K-L), I]]`changes to  `[[I, Z]; [h, * (K-L), I]]` while

```
<SpecialRule>
<Key>AddCommasToMatrices</Key>
<Value>0</Value>
</SpecialRule>
<SpecialRule>
<Key>AddCommasToCellArrays</Key>
<Value>0</Value>
</SpecialRule>
<SpecialRule>
```

Commit b931478, MATLAB 2016b.

**Comment by davidvarga** [comment]
Hi,
unfortunately I am unable to reproduce this behaviour. (Tried on R2016a 64bit).
Could you add more context (maybe the context has some effect on the formatting).

**Comment by homocomputeris** [comment]
Actually, as in the other issue, function call affects the result when using `MBeautify.formatCurrentEditorPage()` _twice_:
`expm(a)*[[I,Z];[h*(K-L),I]]*result;`
`expm(a) * [[I, Z]; [h, * (K-L), I]] * result;`

**Comment by davidvarga** [comment]
Hi!
Okay, "twice" made the magic. I will investigate the problem, thanks for reporting!

**Comment by homocomputeris** [comment]
If there are some spaces already, one time is enough:
```
qCoef = [h; 1 / 6 * h.^3; (1 / 120) * h.^5; (1 / 5040) * h.^7; ...
    (1 / 362880) * h.^9; (1 / 39916800) * h .^ 11];
```
becomes
```
qCoef = [h; 1 / 6 * h .^ 3; (1 / 120) * h .^ 5; (1 / 5040) * h .^ 7; ...
    (1 / 362880) * h .^ 9; (1 / 39916800) * h, .^, 11];
```
And, obviously, this ruins the code.

**Comment by davidvarga** [comment]
I have just added a commit that fixes the first problem (and also the issue with #25). I am investigating the problem with the power operator at the moment.

**Comment by davidvarga** [comment]
I have made an additional commit. These two together should fix this and also #26.
I will close these two issue by the evening. If some of the mentioned examples are still failing on your side, just please let me know!

**Comment by homocomputeris** [comment]
It seems to be working fine, thank you.
